changelog/chore-remove-config-and-qrcode-dependencies

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: dev
+
+chore: remove 'config' and 'qrcode.react' dependencies, since they're not used.
```

Comment:
Can you convert this into a comment? I don't think it makes too much sense as a final changelog entry.